### PR TITLE
Create function for prefix of sticky (for safari).

### PR DESCRIPTION
**Pull request by baptLesta**

### CODE REVIEW

File: workspace/assets/css/core/fct-position.less [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#sticky() {
+	position: sticky;
+	position: -webkit-sticky;
+}
```
Comment (nitriques) [inline workspace/assets/css/core/fct-position.less]: Please add a single empty line at the end of each file !

File: workspace/assets/css/core/fct-position.less [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#sticky() {
+	position: sticky;
+	position: -webkit-sticky;
```
Comment (dominic-blain) [inline workspace/assets/css/core/fct-position.less]: Line 2 & 3 should be switched in order to have the prefixed version first.

Comment (nitriques) [reply workspace/assets/css/core/fct-position.less]: indeed @baptLesta

**Comment by PascalPiche** [comment]
Ping @dominic-blain

**Pull request merged**